src/java/com/phenix/pct/CompilationAttributes.java

```diff
@@ -41,6 +41,8 @@
     private boolean runList = false;
     private boolean listing = false;
     private String listingSource = null;
+    private int pageSize = 55;
```

Comment:
As @stefandrissen said, just use -1 as the default value (which would mean not set).